modules/ROOT/pages/create-community.adoc

```diff
@@ -103,13 +103,45 @@ To verify that Salesforce CRM Content is enabled:
 
 After you create a community, you must create different user profiles for your community _members_ (signed in) and _guests_, so that you can control access to and visibility of APIs and content.
 
+Before configuring member and guest profiles and permissions, open *Setup* > *User Management Settings* and ensure that *Enhanced Profile User Interface* is enabled.
+
 === Configure Member User Profile Permissions
 
-These are the permissions for the member users:
+To configure permissions for the member user profile so your members can access APIs, create client applications, and manage their details:
+
+. In *Setup*, search for *Users* using the *Quick Find* box and click *Users/Profiles*.
+. Click the *Customer Community Plus Login User* profile.
+. Click *Clone*.
+. For *Profile Name* enter `[Site Name] Member User`.
```

Comment:
```suggestion
. For *Profile Name*, enter `[Site Name] Member User`.
```
If [Site Name] is a variable, I think we want to use mono italics rather than [ ]? See style ref. entry for "variables"
```suggestion
. For *Profile Name*, enter `_Site Name_ Member User`.
```

Reply:
I'm sorry for not thinking about this in the previous edit!  :-(

Reply:
Fixed in all instances in this file.